### ISSUE TITLE
thread '<unnamed>' panicked at 'cannot access stderr during shutdown', src\libcore\option.rs:1188:5

### ISSUE THREAD

**Issue by petsuter** [EmbarkStudios/texture-synthesis#144]
Using `texture-synthesis-0.8.0-x86_64-pc-windows-msvc.zip` and the provided example images:
```
C:\texture-synthesis-0.8.0-x86_64-pc-windows-msvc>texture-synthesis.exe --inpaint 1_tile.jpg --out-size 400 --tiling -o out.bmp generate 1.jpg
[00:00:01] ###########################------------- 66%
 stage   6 #############--------------------------- 32%
thread '<unnamed>' panicked at 'cannot access stderr during shutdown', src\libcore\option.rs:1188:5
```

Sometimes it works, sometimes it doesn't.

**Comment by biosmanager** [comment]
I have the same problem, in some cases it finishes but often it crashes with this error.

**Comment by lpil** [comment]
Thanks for the info @biosmanager . Are you also on Windows?

**Comment by biosmanager** [comment]
> Thanks for the info @biosmanager . Are you also on Windows?

Yes.

**Comment by Jake-Shadle** [comment]
https://github.com/rust-lang/rust/issues/29488 seems like it is likely the same issue you are seeing, but I think there is most likely a different a root cause, as you can see from the output the image generation is not actually finished, so we should not be in shutdown, so having a backtrace would be very helpful to point to what exactly is happening.

If you can [install Rust](https://www.rust-lang.org/tools/install) and then clone this repo and do

```sh
set RUST_BACKTRACE=1
cd <directory you cloned to>
cargo run --release -- --inpaint imgs/1_tile.jpg --out-size 400 --tiling -o out.bmp generate imgs/1.jpg
```

that would help us figure out what is going on.

**Comment by Solido** [comment]
Generation never fail when running from cargo.
Replaced the provided release with a fresh build and it is OK 
So I guess you only need to push a new release.
Thanks !

**Comment by Jake-Shadle** [comment]
This should be resolved now in the 0.8.2 release, but please reopen if this is not the case. :slightly_smiling_face:
